common/src/main/java/com/wynntils/features/combat/RangeVisualizerFeature.java

```diff
@@ -120,21 +122,36 @@ private void checkMajorIdCircles(Player player) {
         // Offset the radius slightly so multiple circles can be shown for each player
         // Only a few major IDs can actually be applied at the same time, but we make this general
         List<Pair<CustomColor, Float>> circles = validGear.stream()
-                .flatMap(gearInfo -> gearInfo.fixedStats().majorIds().stream().map(majorId -> switch (majorId.name()) {
-                    case "Taunt" -> Pair.of(CommonColors.ORANGE.withAlpha(TRANSPARENCY), 12f);
-                    case "Saviour’s Sacrifice" -> Pair.of(CommonColors.WHITE.withAlpha(TRANSPARENCY), 8f);
-                    case "Heart of the Pack" -> Pair.of(CommonColors.PINK.withAlpha(TRANSPARENCY), 8.1f);
-                    case "Guardian" -> Pair.of(CommonColors.RED.withAlpha(TRANSPARENCY), 7.9f);
-                    default -> null;
-                }))
+                .flatMap(gearInfo ->
+                        gearInfo.fixedStats().majorIds().stream().map(majorId -> circleFromMajorId(majorId.name())))
                 .filter(Objects::nonNull)
-                .toList();
+                .collect(Collectors.toList());
+
+        // add circles gained from raid major id buffs
+        if (Models.Raid.getCurrentRaid() != null) {
+            for (String buff : Models.Raid.getRaidBuffMajorIds(player.getName().getString())) {
+                Pair<CustomColor, Float> circle = circleFromMajorId(buff);
+                if (circle != null) {
+                    circles.add(circle);
+                }
+            }
+        }
 
         if (!circles.isEmpty()) {
             circlesToRender.put(player, circles);
         }
     }
 
+    private Pair<CustomColor, Float> circleFromMajorId(String majorIdName) {
```

Comment:
```suggestion
    private Pair<CustomColor, Float> getCircleFromMajorId(String majorIdName) {
```

More conventional, although it's more of a nit.

Reply:
Resolved